### PR TITLE
Use the standard header for innkeeper authorization

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: innkeeper/client.go [hunk 1]
```diff
@@ -334,7 +334,7 @@ func (c *Client) writeRoute(url string, route *routeData) error {
 		return err
 	}
 
-	req.Header.Add(authHeaderName, authToken)
+	req.Header.Add(authHeaderName, "Bearer " + authToken)
```
Comment (lmineiro) [inline innkeeper/client.go]: How does innkeeper handle multiple `Authorization` headers? Should be a `.Set()` instead?

Comment (aryszka) [inline innkeeper/client.go]: true

Comment (aryszka) [inline innkeeper/client.go]: updated

**Comment by lmineiro** [comment]
lgtm

**Comment by danpersa** [comment]
lgtm auch

**Pull request merged**